layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+user=$(who am i | awk '{print $1}')
+if (( $EUID != 0 )); then
+    echo "You may need to run this root as sudo if you see permission errors"
```

Comment:
**Minor:**
Could you please change `root` to `script` ?

Reply:
done!